src/modal-container.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import PropTypes from 'prop-types'
+
+const useStyles = makeStyles(() => ({
+  root: {
+    maxWidth: ({ maxWidth }) => maxWidth,
+    width: '80%',
+    background: '#fff',
+    borderRadius: 10,
+    padding: 16,
+    '&:focus': {
+      outline: 'none',
+    },
+  },
+}))
+
+
+const ModalContainer = ({ maxWidth, children }) => {
+  const classes = useStyles({ maxWidth })
+  return (
+    <div className={classes.root}>
+      {children}
+    </div>
+  )
+}
+
+ModalContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+  maxWidth: PropTypes.string,
+}
+
+ModalContainer.defaultProps = {
+  maxWidth: 768,
+}
```

Comment:
Proptype doesn't match number / string? better string maybe user wants to use rem/em/px/% ??

<img width="690" alt="Screen Shot 2021-04-30 at 9 17 18 AM" src="https://user-images.githubusercontent.com/46459367/116700533-e0668080-a994-11eb-9804-4fbb02498f49.png">

Reply:
Good catch, it should be number rather than string